java-client/src/main/java/com/cumulocity/sdk/client/notification/SubscriberImpl.java

```diff
@@ -206,6 +208,14 @@ private void resubscribe(Set<SubscriptionRecord> toResubscribe) {
         removeBrokenListeners();
 
         for (SubscriptionRecord subRec : toResubscribe) {
+            if (!subRec.isAutoRetry()) {
+                String errorMsg = "bayeux client reconnected clientId: " + session.getId();
```

Comment:
Question: is this error message correct? It would mean that the connection is still alive but we are not doing auto handshake/subscribe action?

Reply:
The message is correct, it is being logged (+subscription error handler) on the next cycle of successful handshake/subscribe.